Jenkinsfile

```diff
@@ -34,12 +34,22 @@ pipeline {
                                 try {
                                     checkout scm
                                     sh 'make -f docker.Makefile cross e2e-cross tars'
+                                    sh 'BIN_NAME=yamlschema make -f docker.Makefile cross'
```

Comment:
why having this `yamlschema` separate ? shouldn't `cross` target just create all the binaries (`docker-app` and `yamlschema`)

Reply:
It was far simpler to add a simple line here than modifying cross, which already quite complicated I think...